src/main/scala/mesosphere/marathon/core/task/update/impl/ThrottlingTaskStatusUpdateProcessor.scala

```diff
@@ -0,0 +1,24 @@
+package mesosphere.marathon.core.task.update.impl
+
+import javax.inject.{ Inject, Named }
+
+import mesosphere.marathon.core.task.update.TaskStatusUpdateProcessor
+import mesosphere.util.CapConcurrentExecutions
+import org.apache.mesos.Protos.TaskStatus
+
+import scala.concurrent.Future
+
+object ThrottlingTaskStatusUpdateProcessor {
+  /**
+    * A tag used for dependency injection to disambiguate the dependencies of this processor from
+    * other instances with the same type.
+    */
+  final val dependencyTag = "ThrottlingTaskStatusUpdateProcessor"
+}
+
+private[core] class ThrottlingTaskStatusUpdateProcessor @Inject() (
```

Comment:
(I assumed you mean declaring this constant by "wiring" since you attached the comment to it. The wiring itself, of course, is not done here.)